### ISSUE TITLE
Unsubscribe from all Activities in an ActivityPackage

### ISSUE THREAD

**Issue by wszarmach** [krico/jas#436]
I created an activity package.
I subscribed to it.
I then went and unsubscribed from the individual activities.
The activity package still showed me as a subscriber.
Im not sure how valid a scenario this really is and if it is an actual problem or not.

**Comment by pauljanes** [comment]
Yes I saw same the other day. I think this should be an admin function to clean it up. Because if something gets messed up, but the executions on the AP still shows as 1 (even though the user has been unsubscribed from all individual activities) then it looks weird to the provider. Jayne might think we are not giving her all the money that she is owed. Check the bootcamp june and you will see that there is still one execution, even though i am not subscribed to any activities now). I did exactly as u mentioned - one by one i removed myself from all the individual activities of the AP that i had bought.
